homeassistant/components/motioneye/services.yaml

```diff
@@ -0,0 +1,110 @@
+set_text_overlay:
+  name: Set Text Overlay
+  description: Sets the text overlay for a camera.
+  target:
```

Comment:
Targets are for use with entity services, these services are registered as normal integration services.